markus_test_scripts/examples/usage_example.R

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env Rscript
+
+# Example usage of the Markus RScript testers
```

Comment:
I don't think this file really makes sense. The only purpose of the example scripts is to be uploaded to MarkUs (again, see overall review comment).